### ISSUE TITLE
.deroot() behavior

### ISSUE THREAD

**Issue by balabanmetin** [niemasd/TreeSwift#20]
Hi Niema,

I am using treeswift 1.1.20. Check out the following code:

```
import treeswift as ts
t = ts.read_tree_newick("(A:1,(B:1,C:1):1);")
t.deroot()
print(t)
```

This outputs `(A:1,B:2,C:2);`
Shouldn't  output be `(A:2,B:1,C:1);` ?

**Comment by ekmolloy** [comment]
@niemasd and @balabanmetin - Sorry for late reply! This makes sense to me. My code doesn't utilize the branch lengths, so I don't think that I checked them carefully. Thanks! -Erin

**Comment by niemasd** [comment]
Thank you for letting us know, @ekmolloy !

@balabanmetin Can you update and submit a Pull Request? If not, I can try to tackle this some time in the next few weeks

**Comment by niemasd** [comment]
Finally got around to fixing this over a year later 😄 Sorry for the delay! Fixed in TreeSwift v1.1.32

**Comment by niemasd** [comment]
Woops, actually fixed in v1.1.33 (I forgot to set the `is_rooted` flag to `False` after derooting)
